src/lib/components/collections/req-res-section/sub-components/sub-components-header/Tab.svelte

```diff
@@ -37,7 +51,7 @@
   >
     <button
       on:click={() => {
-        updateCurrentTab(tab.id);
+        onTabSelected(tab.id);
```

Comment:
When the tab is in background, at that time it makes sense to invoke `onTabSelected` event but let's say it is already in foreground, should we still invoke `onTabSelected`?

Reply:
Done, added a condition that it triggers only if tab is not active.